alicloud/connectivity/regions.go

```diff
@@ -76,3 +77,4 @@ var SlbClassicNoSupportedRegions = []Region{APNorthEast1, APSouthEast2, APSouthE
 var NasNoSupportedRegions = []Region{Qingdao, APSouth1, APSouthEast3, APSouthEast5}
 var OssVersioningSupportedRegions = []Region{APSouth1}
 var OssSseSupportedRegions = []Region{Qingdao, Hangzhou, Beijing, Shanghai, Shenzhen, Hongkong, APNorthEast1, APSouth1, USEast1}
+var GpdbClassicNoSupportedRegions = []Region{APSouthEast2, APSouthEast3, APSouthEast5, APSouth1, USEast1, USWest1, APNorthEast1, EUCentral1}
```

Comment:
The supported and unsupported regions should be applied to the acceptance test cases.